<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_prod_dialog</class>
 <widget class="QDialog" name="search_prod_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>129</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>129</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ricerca prodotto</string>
  </property>
  <widget class="QLabel" name="lbl_tutorial">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inserire il nome del prodotto che si desidera ricercare:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_query">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ricerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
